src/pages/jobLogs/PageSizeSelect.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from "@emotion/styled";
```

Comment:
We have a PageSizeSelector component we can likely reuse here.
https://github.com/evergreen-ci/spruce/blob/c8e1c2bdc47363a951790c381cff24e9dd512d32/src/components/PageSizeSelector.tsx